config/config.go

```diff
@@ -165,6 +167,32 @@ func (cfg *AuctionTimeouts) validate(errs []error) []error {
 	return errs
 }
 
+type Compression struct {
+	Request  CompressionConfig `mapstructure:"request"`
+	Response CompressionConfig `mapstructure:"response"`
+}
+
+// CompressionConfig defines if compression is enabled and what type of compression to use
+type CompressionConfig struct {
+	Enabled bool   `mapstructure:"enabled"`
+	CType   string `mapstructure:"type"`
```

Comment:
It makes sense to choose a single response compression, but why do we need to ask for a single request compression method? If PBS support gzip and let's say brotli, why not support both for incoming requests?

Reply:
That's a good point. Will modify the type to be an array